rando_modules/logic.py

```diff
@@ -552,38 +566,39 @@ def _generate_item_pools(
             Item.get(Item.item_name == "PouchE"),
         ]
 
-        cnt_items_removed = 0
+        pool_other_items.extend(pouch_items)
+
+    # Add unused badge duplicates, if needed
+    if add_unused_badge_duplicates:
+        unused_badge_duplicates = []
+        for item in Item.select().where(Item.unused_duplicates == True):
```

Comment:
Not sure of the actual syntax, but surely there's a way to select the items and directly assign them to unused_badge_duplicates without iterating in a for loop?

Reply:
Annoyingly, this is how peewee handles Selects in their ORM, you only get the data you want by iterating over it manually.
For reference see http://docs.peewee-orm.com/en/latest/peewee/querying.html#selecting-multiple-records